core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -764,6 +825,7 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
       (FetchFailed(makeBlockManagerId("hostA"), shuffleId, 0L, 0, 0, "ignored"), null)))
     // this will get called
     // blockManagerMaster.removeExecutor("hostA-exec")
```

Comment:
This comment is not needed as ensured by the `verify`.
```suggestion
```

Reply:
I left the original comment as an explanation for the verify. But I can remove it.